packages/conf-bmake/conf-bmake.1.0/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "1.2"
+maintainer: "michipili@gmail.com"
+homepage: "http://www.crufty.net/help/sjg/bmake.htm"
+authors: "sjg@crufty.net"
+bug-reports: "sjg@crufty.net"
+license: "BSD"
+config-state-vars: [
+  path "The detected path a BSD Make compatible make program."
+]
+build: [
+  ["sh" "%{build}%/files/detect_program.sh" "%{prefix}%/lib/conf-bmake" "bmake"] {!(os = "freebsd" | os = "openbsd" | os = "netbsd")}
+  ["sh" "%{build}%/files/detect_program.sh" "%{prefix}%/lib/conf-bmake" "make"] {(os = "freebsd" | os = "openbsd" | os = "netbsd")}
```

Comment:
Is it reasonable to use `"sh" "-e"` here so that script errors guarantee to fail the detection?